ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis_diag.ecf

```diff
@@ -4,7 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:10:00
-#PBS -l place=vscatter,select=1:mpiprocs=72:ompthreads=1:ncpus=72:mem=48GB
+#PBS -l place=vscatter,select=1:mpiprocs=115:ompthreads=1:ncpus=115:mem=48GB
```

Comment:
Please set mpiprocs=96 to that it is twice mpiprocs in `jenkfgdas_diag.ecf`.

Reply:
Changing back to 96.